### PR TITLE
Feature/responsive menu +  home page

### PR DESCRIPTION
**Pull request by Bluesmile82**
- [Add feature tag for responsive](https://vizzuality.atlassian.net/browse/ORC-393?atlOrigin=eyJpIjoiODU4YWU3NjNlYmY0NDVmN2I4MTc2Y2Q0ZTcyOTg4ZDYiLCJwIjoiaiJ9):  The feature tag is NEXT_PUBLIC_ENABLE_RESPONSIVE
- [Create responsive menu](https://vizzuality.atlassian.net/browse/ORC-392?atlOrigin=eyJpIjoiMWU2MzY2NDAzOGE5NDFhMjhkMTIzMTMzYTYxMGM2NjMiLCJwIjoiaiJ9)
- [Adapt home page to responsive](https://vizzuality.atlassian.net/browse/ORC-391?atlOrigin=eyJpIjoiYjhlM2IyODJhY2I4NGZmZGFkNDQyYjA0NmQ3ZGFmZjMiLCJwIjoiaiJ9)
Mostly lg(1024px) is taken as a reference. Although there are some cases that it adapts to xl(1280px)

### CODE REVIEW

File: client/src/components/home/footer.tsx [hunk 1]
```diff
@@ -25,17 +25,17 @@ const Footer = () => (
         <span className="sr-only">ORCaSa</span>
       </Link>
     </div>
-    <div className="flex w-full items-center justify-between bg-gray-700 px-10 py-8">
-      <div className="flex items-center justify-center gap-4">
+    <div className="flex w-full flex-col items-start justify-between space-y-4 bg-gray-700 px-10 py-8 lg:flex-row lg:items-center lg:space-y-0">
```
Comment (clementprdhomme) [inline client/src/components/home/footer.tsx:28]: The horizontal padding should only be 16px on mobile.

File: client/src/components/mobile-menu/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import { modules, moduleColors } from '@/constants/modules';
+
+const MobileMenu = () => {
+  const [isOpen, setIsOpen] = useMobileMenu();
+  const toggleMenu = () => {
+    setIsOpen(!isOpen);
+  };
+  if (!isOpen) return null;
+  return (
+    <div className="absolute left-0 top-0 z-50 h-screen w-screen transform bg-gray-700 text-white animate-in slide-in-from-right-full">
```
Comment (clementprdhomme) [inline client/src/components/mobile-menu/index.tsx:20]: The focus management of this component is incorrect:
- when the user clicks the hamburger icon, the focus is not moved to this component
- the focus is not trapped within the component

I think that you need to wrap this component in a shadcn/ui dialog and all these issues will probably be fixed.

**Comment by clementprdhomme** [comment]
There are also some improvements you could probably make when looking at the homepage at a 960px size.

File: client/src/components/mobile-menu-bar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import MobileMenuButton from '@/components/mobile-menu-button';
+
+const MobileMenuBar = () => {
+  return (
+    <nav className="z-50 flex items-center justify-between bg-gray-700 px-6 py-4 text-white lg:hidden">
```
[the remark this answers is not in the document]
Comment (Bluesmile82) [reply client/src/components/mobile-menu-bar/index.tsx:8]: Sorry, which horizontal space? The px is 6 here = 24px

File: client/src/components/home/modules-section.tsx [hunk 4]
```diff
@@ -236,12 +257,12 @@ const ModulesSection = () => (
         </div>
       </div>
     </div>
-    <div className="relative flex min-h-[616px] items-center justify-center">
+    <div className="relative flex min-h-[1040px] items-center justify-center lg:min-h-[616px]">
       <div className="absolute top-0">
-        <img src="/images/map.png" className="h-[616px] w-full" alt="" />
+        <img src="/images/map.png" className="w-[125%] lg:h-[616px] lg:w-full" alt="" />
```
[the remark this answers is not in the document]
Comment (Bluesmile82) [reply client/src/components/home/modules-section.tsx:262]: image
I can see it on mobile. Is not seen in the whole module. But that is like the designs. Do you have an example (w,h)?

File: client/src/components/home/carousel.tsx [hunk 5]
```diff
@@ -123,6 +130,16 @@ const Carousel = () => {
           <span className="sr-only">Next testimony</span>
         </Button>
       </div>
+      {/* For mobile */}
+      <CarouselComponent className="block lg:hidden">
+        <CarouselContent>
+          {cards.map((card) => (
+            <CarouselItem key={card.id} className="flex justify-center shadow">
+              <Card card={card} />
```
[the remark this answers is not in the document]
Comment (Bluesmile82) [reply client/src/components/home/carousel.tsx:138]: The background seems to be white as seen when you play the prototype

**Pull request merged**